tests/test_opener.py

```diff
@@ -128,3 +128,45 @@ def save_predictions(self, y_pred, path):
 
     o.data_folders = data_paths
     assert o.get_X() == 'Xclass'
+
+
+@pytest.mark.parametrize('save_predictions_method_body', (
```

Comment:
I'm wondering if this test could be written without importing the opener class as a module from a docstring?

Reply:
Probably but I didn't want to depart from the style of all other tests in this test file. Should I try to not use a string?

Reply:
Yes that's my fault. For the `load_*` tests it makes sense to use an import but for other tests I don't think (test_opener_check_folders).

Yes you could try to define the opener directly and check if it simplifies the test.